install/update_94_95.php

```diff
@@ -1657,6 +1657,25 @@ function update94to95() {
    }
    /** /Update default right assignement rule */
 
+   if (!$DB->fieldExists('glpi_profiles', 'managed_domainrecordtypes')) {
+      $migration->addField(
+         'glpi_profiles',
+         'managed_domainrecordtypes',
+         'text',
+         [
+            'after'     => 'change_status'
```

Comment:
```suggestion
            'after'     => 'change_status',
            'value'    => exportArrayToDB([]),
```

Reply:
Nope, default value on a TEXT field will fail on travis (it sems that happens on some environments, I do not know why).
See https://travis-ci.org/trasher/glpi/jobs/650775602